src/main/java/uk/gov/ons/census/casesvc/model/dto/RefusalDTO.java

```diff
@@ -10,8 +10,15 @@
 @JsonInclude(Include.NON_NULL)
 public class RefusalDTO {
   private RefusalTypeDTO type;
-  private String report;
   private String agentId;
   private String callId;
+  private boolean isHouseholder;
+
+  public boolean getIsHouseholder() {
```

Comment:
Can you put a comment here to say that this is needed otherwise we will get `householder` in the JSON instead of `isHouseholder` pls